standard/examples/examples_subsetting.adoc

```diff
@@ -1,6 +1,6 @@
 
-*   http://acme.com/oapi/collections/%7dcoverageid%7d/coverage/rangeset?subset=Lat(40,50)&subset=Long(10,20)["http://acme.com/oapi/collections/{coverageid}/coverage/rangeset?subset=Lat(40,50)&subset=Long(10,20)"] -- returns a coverage cutout between (40,10) and (50,20), in the coverage's Native Format
-*   http://acme.com/oapi/collection/%7bcoverageid%7d/coverage/rangeset?subset=time(2019-03-27)["http://acme.com/oapi/collections/{coverageid}/coverage/rangeset?subset=time(\"2019-03-27\")"] -- returns a coverage slice at the timestamp given, with all data available in the other dimensions (in case the coverage is Lat/Long/time the result will be a 2D image with its full Lat/Lon extent)
-*   http://acme.com/oapi/collections/%7Bcoverageid%7D/coverage?subset=Lat(40,50)&subset=Long(10,20)["http://acme.com/oapi/collections/{coverageid}/coverage?subset=Lat((40,50)&subset=Long(10,20)"] -- returns a cutout from the coverage identified with extent between corner coordinates (40,10) and (50,20) (note: replace the _ in the URL by a comma character - an adoc issue)
-*   http://acme.com/oapi/collections/%7Bcoverageid%7D/coverage/rangeset?subset=Lat(40,50)&subset=Long(10,20)["http://acme.com/oapi/collections/{coverageid}/coverage/rangeset?subset=Lat(40,50)&SUBSET=Long(10,20)"]  -- returns the range set of a coverage cutout between (40,10) and (50,20), in the coverage’s Native Format; no domain set, range type, and metadata will be returned (note: replace the _ in the URL by a comma character - an adoc issue)
-*   http://acme.com/oapi/collections/%7Bcoverageid%7D/coverage?subset=time(%222019-03-27%22)["http://acme.com/oapi/collections/{coverageid}/coverage?subset=time(\"2019-03-27\")"] -- returns a coverage slice at the timestamp given (in case the coverage is Lat/Long/time the result will be a 2D image)
+* {datasetAPI}/collections/{coverageid}/coverage/rangeset?subset=Lat(40:50),Lon(10:20) -- returns a coverage cutout (sample values only) between (Lat: 40, Lon: 10) and (Lat: 50, Lon: 20), in the negotiated format
```

Comment:
Could be deleted as it is the same example as the last one, isn't it?

Reply:
Is it about `/coverage/rangeset` vs. `/coverage` ?